commands/miscellaneous/getuserids.js

```diff
@@ -1,46 +1,31 @@
-const { Command } = require("discord.js-commando");
 const Discord = require('discord.js');
+const { SlashCommandBuilder } = require('@discordjs/builders');
 
-module.exports = class GetUserIdsCommand extends Command {
-  constructor(client) {
-    super(client, {
-      name: "getuserids",
-      aliases: ["getids"],
-      group: "miscellaneous",
-      memberName: "getuserids",
-      description: "Gets a list of user ids that were mentioned in a message.\n Usage:getuserids [messageID] [channelID]",
-      userPermissions: [],
-      args: [
-        {
-          key: "messageIDs",
-          prompt: "What messages would you like to get the user ids from?",
-          type: "string",
-        },
-        {
-            key: "channel",
-            prompt: "In what channel is this message?",
-            type: "channel",
-        },
-      ],
-    });
-  }
+module.exports = {
+    data: new SlashCommandBuilder()
+        .setName('getuserids')
+        .setDescription('Gets a list of user ids that were mentioned in a message')
```

Comment:
not sure if you left it out intentionally, but this is missing the `\n Usage:getuserids [messageID] [channelID]` part from the original description

Reply:
oh wait, does the autocomplete show the usage? I haven't written any slash commands with actual args yet 😆

Reply:
when you enter a slash command with args, you hit tab after each one and it will show you the arg and the description. i will steal natalies bot to make examples since mine isnt running. 😛 

base command
<img width="329" alt="Screen Shot 2021-11-16 at 17 22 37" src="https://user-images.githubusercontent.com/17508245/142075684-9bc4e3f3-2f30-4611-bc99-029715507565.png">

example with an arg and description
<img width="257" alt="Screen Shot 2021-11-16 at 17 22 31" src="https://user-images.githubusercontent.com/17508245/142075848-666df716-130e-4cff-9f5e-e40d38f2afdb.png">